internal/app/machined/pkg/controllers/k8s/kubelet_spec.go

```diff
@@ -109,8 +109,7 @@ func (ctrl *KubeletSpecController) Run(ctx context.Context, r controller.Runtime
 		expectedNodename := nodenameSpec.Nodename
 
 		args := argsbuilder.Args{
-			"container-runtime-endpoint": "unix://" + constants.CRIContainerdAddress,
```

Comment:
migrated deprecated flag to kubelet config option (it's available for 16 months now)